README.md

```diff
@@ -1,2 +1,181 @@
-# code_review
-In this repo i will review my own code "Portfolio mobile first" project
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
```

Comment:
kindly remove commented lines from your README.
It may not seem like a big change for anyone reading it but in the future it will save you some scrolling and unnecessary clutter! Making your code look cleaner will improve it's readability. ☺